packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -54,6 +71,36 @@ export function hasFinishedResolution( state, selectorName, args = [] ) {
 	return getIsResolving( state, selectorName, args ) === false;
 }
 
+/**
+ * Returns true if resolution has failed for a given selector
+ * name, and arguments set.
+ *
+ * @param {State}     state        Data state.
+ * @param {string}    selectorName Selector name.
+ * @param {unknown[]} [args]       Arguments passed to selector.
+ *
+ * @return {boolean} Has resolution failed
+ */
+export function hasLastResolutionFailed( state, selectorName, args = [] ) {
```

Comment:
Should we drop the "last" from the name of these selectors. I mean I see the value in clarity but it's a departure from isResolving which is also about the "last resolution"

Reply:
Great idea! I just updated the PR. I borrowed this naming from `getLastEntityDeleteError` and `getLastEntitySaveError` in `core-data`. I wonder if we should deprecate them in favor of `getEntityDeleteError` and `getEntitySaveError`? Either way, that's another discussion.